src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs

```diff
@@ -375,6 +383,214 @@ public Range ExposureRange
         }
         #endregion Exposure
 
+        #region saturation
+        /// <summary>
+        /// Gets or sets the camera saturation value.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        /// <exception cref="ObjectDisposedException">The camera has already been disposed. </exception>
+        public int Saturation
+        {
+            get
+            {
+                Native.GetSaturation(_camera.GetHandle(), out int val).
+                    ThrowIfFailed("Failed to get camera saturation value");
+
+                return val;
+            }
+            set
+            {
+                Native.SetSaturation(_camera.GetHandle(), value).
+                    ThrowIfFailed("Failed to set camera saturation value.");
+            }
+        }
+
+        /// <summary>
+        /// Gets the available saturation value.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        /// <remarks>
+        /// If the minimum value is greater than the maximum value, it means this feature is not supported.
+        /// </remarks>
+        /// <exception cref="NotSupportedException">Saturation is not supported.</exception>
```

Comment:
lines 413 and 415 suggest 2 different outcomes when saturation is not supported. Which is it then - an empty range or exception?

Reply:
As described in remarks, If min value is greater than the max value, it means XXRange is not supported and in this case, NotSupportedException will be thrown. 
Those values are tested by `GetRange()` method in 64 line.

Reply:
The thing is that it's not clear to me what will happen when the feature is not supported - It cannot be both an exception and some values.  Will app developer get an exception or an empty Range object?